test/passes/roundtrip.txt

```diff
@@ -6,6 +6,6 @@
  )
 )
 (module
- (memory $ 1 1)
- (table $ 0 funcref)
+ (memory $0 1 1)
+ (table $0 0 funcref)
```

Comment:
I have no idea why this changed but it looks much more correct this way.

Reply:
Oh, probably because the memory and table weren't being reinitialized when the module was cleared before.